src/features/authorSlice.ts

```diff
@@ -0,0 +1,31 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+
+type AuthorState = {
+  author: User | null;
+  loading: boolean;
+  hasError: boolean;
+};
+
+const initialState: AuthorState = {
+  author: null,
+  loading: false,
+  hasError: false,
+};
+
+const authorSlice = createSlice({
+  name: 'user',
+  initialState,
+  reducers: {
+    set: (state, action: PayloadAction<User>) => {
+      state.author = action.payload;
```

Comment:
disable this lint rule in "slice" files
![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/86923089/c6c3c1dd-b715-4eb9-8db0-0555996de19d)